SportsHub/SportsHub.DAL/Repository/ArticleRepository.cs

```diff
@@ -17,5 +17,11 @@ public ArticleRepository(ApplicationDbContext context) : base(context)
                 .Where(x => x.Title == title)
                 .FirstOrDefaultAsync();
         }
+
+        public async Task DeleteArticleAsync(string title)
+        {
+            Article? article = await GetByTitleAsync(title);
```

Comment:
Repository level should not contain any logic. Here you have duplication calls to the DB, you already have called this in the Service layer. Why don`t use that result?

Reply:
If you have a delete action in the DB, so do deletion only. And add in parameter item that should be deleted

Reply:
So i need Dto . Because i though with this method we already had it wasn't needed ti create a DeleteArticleDto

Reply:
implemented it